lighthouse-core/lib/i18n.js

```diff
@@ -66,6 +67,21 @@ const formats = {
   },
 };
 
+/**
+ * Look up the best available locale for the requested language, falling back by
+ * looking at progressively shorter prefixes (`de-CH-1996` -> `de-CH` -> `de`),
+ * and finally returning the default locale ('en-US') if no match is found.
+ * @param {string=} locale
+ * @return {LH.Locale}
+ */
+function lookupLocale(locale = MessageFormat.defaultLocale) {
```

Comment:
`MessageFormat.defaultLocale` is actually `en` when no locale info is found, not  `en-US`. Not sure if we have an issue with that since we assume it's `en-US` everywhere else. As a result, we'll only get the `en-US` default set at the end of this function if a specific locale we don't support is requested. Do we want to change the last line to return `en` instead? Or just live with this

Reply:
these words wash over me like an ocean wave. 🌊 i don't know what hit me but it was powerful and disorienting.  

😕

Reply:
> these words wash over me like an ocean wave

haha. Basically everywhere we pretend the default has been `en-US`, but the current `i18n.getDefaultLocale` has been trying to return `en` since that's what the default fallback of `MessageFormat.defaultLocale` is. It just then finds out there is no `en` strings file, so the function falls back to the hardcoded `en-US`. Once we have an `en` strings file,  that will change.

So,
- if you request whatever the default is for where LH is running, you'll get `en`
- if you request a locale that we don't have (and have no fallback for), you'll get `en-US`

I'm wondering if we should
- change default fallback everywhere to `en` to match `MessageFormat`
- change `MessageFormat.defaultLocale` so that if it returns `en`, to be `en-US` and not change everywhere else
- live with the inconsistency